src/components/Button.js

```diff
@@ -2,14 +2,35 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 export default function Button(props) {
-  const { name } = props;
+  const { color, wide, name } = props;
+  const style = {};
+
+  (function isWide() {
+    if (wide) {
+      style.width = '50%';
+    }
+  }());
+
+  (function isOrange() {
```

Comment:
- [ ] The default button color should be orange:

![](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/colorProps.png)